src/rpc/misc.cpp

```diff
@@ -449,6 +451,36 @@ static UniValue getmemoryinfo(const JSONRPCRequest& request)
     }
 }
 
+static UniValue getgeneralinfo(const JSONRPCRequest& request)
+{
+    RPCHelpMan{"getgeneralinfo",
+                "\nReturns data about the bitcoin daemon.\n",
+                {},
+                RPCResult{
+                "  {\n"
+                "    \"clientversion\": \"string\",     (string) Client version\n"
+                "    \"useragent\":\"string\",          (string) Client name\n"
+                "    \"datadir\":\"path\",              (string) Data directory path\n"
+                "    \"blocksdir\":\"path\",            (string) Blocks directory path\n"
+                "    \"startuptime\":\"datetime\",      (string) Startup time\n"
+                "  }\n"
+                },
+                RPCExamples{
+                    HelpExampleCli("getgeneralinfo", "")
+            + HelpExampleRpc("getgeneralinfo", "")
+                },
+            }.Check(request);
+
+        UniValue obj(UniValue::VOBJ);
+        obj.pushKV("clientversion", FormatFullVersion());
+        obj.pushKV("useragent", strSubVersion);
+        obj.pushKV("datadir", GetDataDir().string());
+        obj.pushKV("blocksdir", GetBlocksDir().string());
+        obj.pushKV("startuptime", FormatISO8601DateTime(GetStartupTime()));
```

Comment:
Times should be seconds-since-epoch Numbers, not formatted strings.

Reply:
I tried to imitate the output in the UI, hence the formatting. However, this is no problem, I can return it as seconds-since-epoch. But I think this PR will never get into the code anyway (please, check previous discussion if not done already).

Reply:
Have updated the code. Thanks for the hint.